runtime/ts/slot-composer.ts

```diff
@@ -116,16 +116,22 @@ export class SlotComposer {
         }
 
         let slotConsumer = this.consumers.find(slot => slot instanceof HostedSlotConsumer && slot.hostedSlotId === cs.targetSlot.id);
+        let parentTransformation = null;
```

Comment:
`parent` is a new terminology here that isn't used anywhere else, also `parentTransformation` doesn't imply what type of object that is - consumer? context? particle? maybe just stick with transformationSlotConsumer - it is indeed very long, but self explanatory?

Reply:
Done.